docs/adr/000x-modelling-history.md

```diff
@@ -0,0 +1,250 @@
+produces multiple files that are uploaded to Asset Manager for different sizing
+variations. Images are modelled in a similar way to Revision with an
+Image::Revision model, as represented below:
+
+![Image Revision breakdown](000x/image-revision-diagram.png)
+
+The Image model itself is used for continuation between image revisions. It is
+known that two Image::Revisions are versions of the same item if they share the
+same Image association.
+
+The data of an Image::Revision is stored between an Image::FileRevision and an
+Image::MetadataRevision. These two objects differ by the fact that any change to
+Image::FileRevision requires changes to the resultant Asset Manager files (such
+as crop dimensions), whereas Image::MetadataRevision stores accompanying data
+that doesn't affect the Asset Manager files (such as alt text).
+
+Image::FileRevision is associated with an ActiveStorage::Blob object that is
+responsible for managing the storage of the source file. It also has a one to
+many association with Image::Asset, each of these represent resulting files that
+are uploaded to Asset Manager. The Image::Asset model stores the URL to the
+Asset Manager file and what state the file is on Asset Manager.
+
+### Timeline
+
+The TimelineEntry model represents an event that should be shown to a user as
+part of a visual timeline of a document's history. It stores the event which
+occurred and associations to relevant models. The sole purpose of this model
+is for representing a timeline to users visually and not for wider historical
+uses outside of the timeline view.
+
+The TimelineEntry model should not store data which could not be
+derived from other aspects of a document. This is to maintain
+flexibility for timeline to be an evolving and iterated part of the
+application. Previously models such as Removal and Withdrawal were associated
+directly with a TimelineEntry which led to using this model outside of
+displaying a timeline. These are now suggested to be associated with a Status
+model.
+
+At the time of writing it wasn't yet determined what the
+timeline would show, and therefore it wasn't clear exactly how
+best to model them. Because of this TimelineEntry will be modelled in a
+speculative way with a number of references to relevant data.
+
+### Topics
+
+No data related to topics (otherwise known as GOV.UK taxonomy) is intended to
+be stored in Content Publisher at the current point in time. This is due to
+topics being accessed and edited by directly interacting with the Publishing
+API. Other applications, notably Content Tagger, can also edit topics which has
+the consequence that the Publishing API is the source of truth for this data
+rather than Content Publisher.
+
+This inconsistency makes it difficult to store the history of topics in a
+reliable way. Thus, until needs are determined to store this data, Content
+Publisher will continue to query the Publishing API rather than storing this
+data.
+
+### Overview
```

Comment:
Yeah summary has the same problem has overview in that it's frequently used at the introduction rather than conclusion of a document. I purposely opted out of summary since that carried connotations of brief and simplified since this is probably the most complicated part.

For lack of a better term I've gone with "Collated Diagram" which may well be the worst of all worlds. If you have any other suggestions that'd be great. I'm looking for something that reflects that it's the combined picture which I was what I hoped "Overview" portrayed.